app/src/ui/toolbar/push-pull-button.tsx

```diff
@@ -32,6 +33,9 @@ interface IPushPullButtonProps {
 
   /** The current repository */
   readonly repository: Repository
+
+  /** The current state of the tip of the repository */
+  readonly tipState: TipState
```

Comment:
I'm not sure if this documentation adds much. Could we add a note about what it's used for? I.e. to disable the push action when the current branch is unborn?